.env.test

```diff
@@ -8,6 +8,7 @@ SLACK_BOT_ACCESS_TOKEN=test_slack_bot_access_token
 SLACK_AUTH_TOKEN_HASH=test_slack_auth_token_hash
 SLACK_SIGNING_SECRET=test_slack_signing_secret
 ADMIN_CONTROL_ROOM_SLACK_CHANNEL_ID=test_admin_control_room_slack_channel_id
+TEAM_ID=test_slack_team_id
```

Comment:
Can you add how to get this to https://docs.google.com/document/d/1_zAwkDKnpcphb9hC2Oro4M-rdXkLLqpFnh_Nt5I19Sk/edit#heading=h.akt7k744o5cn ? And also make sure to ping Andrew/Tomer after this is merged so they can update their dev envs